config/pokeapi.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import (
+	"fmt"
+	"go-bootcamp/usecase/clients"
+	"log"
+
+	"github.com/spf13/viper"
+)
+
+var PokeAPI clients.PokeAPIConfig
```

Comment:
Document all the variables functions, interfaces, constants that are being exported